docs/_articles/en/v3.x/scanner-commands/run.md

```diff
@@ -149,3 +149,29 @@ $ sfdx scanner:run --target "/some-project/" --severity-threshold 2
 
 ## Demo
 ![Run Example](./assets/images/run.gif)
+
+## Target Patterns
+
+In `~/.sfdx-scanner/{{ site.data.versions-v3.configfile }}`, you can specify target patterns for each engine. 
+
+If you are using ESLint and ESLint-LWC and want to avoid duplicate violations, replace `"**/*.js"` with the pattern for your LWC files to the `eslint-lwc` `targetPatterns` and add the negative pattern to the `eslint` `targetPatterns` . For example, if your LWC files are stored in a directory called `lwc`, your `{{ site.data.versions-v3.configfile }}` file might include this information:
+```
+{
+    "name": "eslint",
+    "targetPatterns": [
+        "**/*.js",
+        "!**/lwc/**.js",
+        "!**/node_modules/**",
+        "!**/bower_components/**"
+    ],
+    "disabled": false
+},
+{
+    "name": "eslint-lwc",
+    "targetPatterns": [
+        "**/lwc/**.js",
+        "!**/node_modules/**"
+    ],
+    "disabled": true
+},
+```
```

Comment:
Should we add specifics about formatting a target pattern (** matches any number of characters..., etc) or can we assume users will know / be able to interpret?

Reply:
I think we can just tell them that it's a glob, and that it uses standard glob syntax. If they don't already know about globs, it's easy enough to google.

Reply:
Sounds good